pkg/advisory/create.go

```diff
@@ -32,12 +32,15 @@ func Create(ctx context.Context, req Request, opts CreateOptions) error {
 		return createAdvisoryConfig(ctx, opts.AdvisoryDocs, req)
 
 	case 1:
-		newAdvisoryID := req.VulnerabilityID
-
 		// i.e. exactly one advisories file for this package
 		u := v2.NewAdvisoriesSectionUpdater(func(doc v2.Document) (v2.Advisories, error) {
-			if _, exists := doc.Advisories.Get(newAdvisoryID); exists {
-				return v2.Advisories{}, fmt.Errorf("advisory %q already exists for %q", newAdvisoryID, req.Package)
+			if _, exists := doc.Advisories.GetAlias(req.Aliases[0]); exists {
+				return v2.Advisories{}, fmt.Errorf("advisory %q already exists for %q", req.Aliases[0], req.Package)
+			}
```

Comment:
I don't think we want to assign special meaning to `req.Aliases[0]` with this new way of doing things. The CGA ID should be the definitive identifier for an advisory; and in contexts like this one ☝️ where the caller doesn't yet know the CGA ID, all aliases (that is, CVE and GHSA IDs for now) should carry equal wait, and we should assume each member of the alias set is equally useful for determining if an advisory already exists to describe this given vulnerability.

Reply:
reverted